standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/RawStore.java

```diff
@@ -1588,10 +1606,22 @@ void getFileMetadataByExpr(List<Long> fileIds, FileMetadataExprType type, byte[]
    * matches the arguments the results here will be all mixed together into a single list.
    * @throws MetaException error access the RDBMS.
    */
+  @Deprecated
   List<SQLForeignKey> getForeignKeys(String catName, String parent_db_name,
     String parent_tbl_name, String foreign_db_name, String foreign_tbl_name)
     throws MetaException;
 
+  /**
+   * Get the foreign keys for a table.  All foreign keys for a particular table can be fetched by
+   * passing null for the last two arguments.
```

Comment:
What do you mean by this ? - "All foreign keys for a particular table can be fetched by
    * passing null for the last two arguments." . You are only passing one argument here.

Reply:
Done